src/main/java/uk/nhs/hee/tis/trainee/forms/api/ConditionsOfJoiningResource.java

```diff
@@ -49,9 +50,11 @@
 public class ConditionsOfJoiningResource {
 
   private final PdfService pdfService;
+  private final TraineeIdentity traineeIdentity;
 
-  public ConditionsOfJoiningResource(PdfService pdfService) {
+  public ConditionsOfJoiningResource(PdfService pdfService, TraineeIdentity traineeIdentity) {
```

Comment:
The forms and LTFT resources have the trainee identity injected into the service, rather than into the resource. Since the CoJ doesn't really have a dedicated service, the trainee identity is injected directly into the resource, but this inconsistency might be worth ironing-out.

Reply:
That's sensible enough to keep the scope of this PR from creeping too far. Do you want to put a TODO here for that refactoring so we can track it via Sonar?